test/column.test.js

```diff
@@ -0,0 +1,83 @@
+import { compressors } from 'hyparquet-compressors'
+import { describe, expect, it } from 'vitest'
+import { parquetMetadata } from '../src/hyparquet.js'
+import { getSchemaPath } from '../src/schema.js'
+import { getColumnRange, readColumn } from '../src/column.js'
+import { asyncBufferFromFile } from '../src/utils.js'
+
+describe('readColumn', () => {
+  it('read columns when rowLimit is undefined', async () => {
```

Comment:
we could factor these four tests with https://vitest.dev/api/#test-for, as they only differ in the last two lines

Reply:
https://github.com/hyparam/hyparquet/pull/55